src/server/pachyderm_test.go

```diff
@@ -5397,69 +5416,70 @@ func TestCronPipeline(t *testing.T) {
-
-		// subscribe to the pipeline1 cron repo and wait for inputs
-		ctx, cancel := context.WithTimeout(context.Background(), time.Second*120)
-		defer cancel() //cleanup resources
-		iter, err := c.WithCtx(ctx).SubscribeCommit(pipeline, "master", "", pfs.CommitState_STARTED)
-		require.NoError(t, err)
-		for i := 0; i < 5; i++ {
-			commitInfo, err := iter.Next()
-			require.NoError(t, err)
-			_, err = c.BlockCommit(commitInfo.Commit.Repo.Name, commitInfo.Commit.ID)
-		}
-	})
+	// Skipping incremental for 1.8.0
+	//t.Run("CronIncremental", func(t *testing.T) {
+	//	pipeline := tu.UniqueString("CronIncremental-")
+	//	req := &pps.CreatePipelineRequest{
+	//		Pipeline: &pps.Pipeline{Name: pipeline},
+	//		Transform: &pps.Transform{
+	//			Cmd: []string{"bash"},
+	//			Stdin: []string{
+	//				"cat /pfs/time/time >> /pfs/out/time",
+	//				"echo \"\" >> /pfs/out/time",
+	//			},
+	//		},
+	//		Input:       client.NewCronInput("time", "@every 10s"),
+	//		Incremental: true,
+	//	}
+	//	_, err := c.PpsAPIClient.CreatePipeline(c.Ctx(), req)
+	//	require.NoError(t, err)
+
+	//	// subscribe to the pipeline1 cron repo and wait for inputs
+	//	ctx, cancel := context.WithTimeout(context.Background(), time.Second*120)
+	//	defer cancel() //cleanup resources
+	//	iter, err := c.WithCtx(ctx).SubscribeCommit(pipeline, "master", "", pfs.CommitState_STARTED)
+	//	require.NoError(t, err)
+	//	for i := 0; i < 5; i++ {
+	//		commitInfo, err := iter.Next()
+	//		require.NoError(t, err)
+	//		_, err = c.BlockCommit(commitInfo.Commit.Repo.Name, commitInfo.Commit.ID)
+	//		require.NoError(t, err)
+	//		var buf bytes.Buffer
+	//		require.NoError(t, c.GetFile(commitInfo.Commit.Repo.Name, commitInfo.Commit.ID, "time", 0, 0, &buf))
+	//		require.Equal(t, i+2, len(strings.Split(buf.String(), "\n")))
+	//	}
+	//})
+
+	//t.Run("CronIncrementalFailures", func(t *testing.T) {
+	//	pipeline := tu.UniqueString("CronIncremental-")
+	//	req := &pps.CreatePipelineRequest{
+	//		Pipeline: &pps.Pipeline{Name: pipeline},
+	//		Transform: &pps.Transform{
+	//			Cmd: []string{"bash"},
+	//			Stdin: []string{
+	//				"FLIP=$(($(($RANDOM%10))%2))",
+	//				"if [ $FLIP -eq 0 ]; then exit 1; fi",
+	//				"cat /pfs/time/time >> /pfs/out/time",
+	//				"echo \"\" >> /pfs/out/time",
+	//			},
+	//		},
+	//		Input:       client.NewCronInput("time", "@every 10s"),
+	//		Incremental: true,
+	//	}
+	//	_, err := c.PpsAPIClient.CreatePipeline(c.Ctx(), req)
+	//	require.NoError(t, err)
+
+	//	// subscribe to the pipeline1 cron repo and wait for inputs
+	//	ctx, cancel := context.WithTimeout(context.Background(), time.Second*120)
+	//	defer cancel() //cleanup resources
+	//	iter, err := c.WithCtx(ctx).SubscribeCommit(pipeline, "master", "", pfs.CommitState_STARTED)
+	//	require.NoError(t, err)
+	//	for i := 0; i < 5; i++ {
+	//		commitInfo, err := iter.Next()
+	//		require.NoError(t, err)
+	//		_, err = c.BlockCommit(commitInfo.Commit.Repo.Name, commitInfo.Commit.ID)
+	//	}
+	//})
```

Comment:
Can this be skipped like other tests? If you're going to comment out a test you almost always might was well remove it. It'll stop compiling real quick.

Reply:
This will be removed when I commit the removal of incremental.